packages/types/src/board.ts

```diff
@@ -94,3 +97,10 @@ export interface ISearchBoardRequestParams {
 export interface ISearchBoardResponseBody {
   boards: IBoardPrivate[];
 }
+
+export interface getBoardQueryOptions {
```

Comment:
@TaranaKhanna You can refer to the other interface type definitions for the naming conventions.